tap_bigquery/sync_bigquery.py

```diff
@@ -219,12 +222,12 @@ def do_sync(config, state, stream):
                 elif prop.type[1] == "integer" and row[key]:
                     record[key] = int(row[key])
                 else:
-                    record[key] = row[key]
+                    record[key] = r.isoformat() if r else None
```

Comment:
I believe `r` is not defined here. Its scope is only under the if-block starting Ln 200. Plus, `r.isofomrat()` assumes r is date or datetime type, which may not be the case outside your use case...I suggest leaving this line as before. Thoughts?

Reply:
I mis-copied this 🤦‍♂️ good spot. Let me know if looks ok now?